lib/Runtime/Library/JavascriptProxy.h

```diff
@@ -147,7 +147,6 @@ namespace Js
 
         // proxy does not support IDispatch stuff.
         virtual Var GetNamespaceParent(Js::Var aChild) { AssertMsg(false, "Shouldn't call this implementation."); return nullptr; }
```

Comment:
Should we remove this one as well?

Reply:
That seems reasonable, since `RecyclableObject` has the same `return nullptr` behavior, and nothing else overrides it.